docs/hub/datasets-fiftyone.md

```diff
@@ -0,0 +1,144 @@
+
+```bash
+huggingface-cli login
+```
+
+Make sure you have `fiftyone>=0.24.0` installed:
+
+```bash
+pip install -U fiftyone
+```
+
+## Loading Visual Datasets from the Hub
+
+With `load_from_hub()` from FiftyOne's Hugging Face utils, you can load:
+
+- Any FiftyOne dataset uploaded to the hub
+- Most image-based datasets stored in Parquet files (which is the standard for datasets uploaded to the hub via the `datasets` library)
+
+### Loading FiftyOne datasets from the Hub
+
+Any dataset pushed to the hub in one of FiftyOne’s [supported common formats](https://docs.voxel51.com/user_guide/dataset_creation/datasets.html#supported-import-formats)
+should have all of the necessary configuration info in its dataset repo on the
+hub, so you can load the dataset by specifying its `repo_id`. As an example, to
+load the [VisDrone detection dataset](https://huggingface.co/datasets/Voxel51/VisDrone2019-DET),
+all you need is:
+
+```python
+import fiftyone as fo
+from fiftyone.utils import load_from_hub
+
+## load from the hub
+dataset = load_from_hub("Voxel51/VisDrone2019-DET")
+
+## visualize in app
+session = fo.launch_app(dataset)
+```
+
+![FiftyOne VisDrone dataset](https://cdn-uploads.huggingface.co/production/uploads/63127e2495407887cb79c5ea/0eKxe_GSsBjt8wMjT9qaI.jpeg)
+
+You can [customize the download process](https://docs.voxel51.com/integrations/huggingface.html#configuring-the-download-process), including the number of samples to
+download, the name of the created dataset object, whether or not it is persisted
+to disk, and more!
+
+You can list all the available FiftyOne datasets on the Hub using:
+
+```python
+from huggingface_hub import HfApi
+api = HfApi()
+api.list_datasets(tags="fiftyone")
+```
+
+### Loading Parquet Datasets from the Hub with FiftyOne
+
+You can also use the `load_from_hub()` function to load datasets from Parquet
+files. Type conversions are handled for you and images are downloaded from URLs
+if necessary.
+
+With this functionality, [you can load](https://docs.voxel51.com/integrations/huggingface.html#basic-examples):
+
+- Image classification datasets like [Food101](https://huggingface.co/datasets/food101) and [ImageNet-Sketch](https://huggingface.co/datasets/imagenet_sketch)
+- Object detection datasets like [CPPE-5](https://huggingface.co/datasets/cppe-5) and [WIDER FACE](https://huggingface.co/datasets/wider_face)
+- Segmentation datasets like [SceneParse150](https://huggingface.co/datasets/scene_parse_150) and [Sidewalk Semantic](https://huggingface.co/datasets/segments/sidewalk-semantic)
+- Image captioning datasets like [COYO-700M](https://huggingface.co/datasets/kakaobrain/coyo-700m) and [New Yorker Caption Contest](https://huggingface.co/datasets/jmhessel/newyorker_caption_contest)
+- Visual question-answering datasets like [TextVQA](https://huggingface.co/datasets/textvqa) and [ScienceQA](https://huggingface.co/datasets/derek-thomas/ScienceQA)
+  And many more!
+
```

Comment:
Not required for this PR but it could be worth making a Collection for datasets that are FiftyOne compatible.

Reply:
Great suggestion!